### ISSUE TITLE
GroupedList: Right/left arrow no longer expand and collapse groups

### ISSUE THREAD

**Issue by betrue-final-final** [microsoft/fluentui#6318]
Mac OSX High Sierra, Chrome browser
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)
 OneDrive, SharePoint, etc...lots

There's no way to expand or collapse a group with keyboard.
Expected: With focus on the group header, right or left arrow should expand or collapse the group.

**Comment by micahgodbolt** [comment]
This wasn't working as far back as 6.49 (the first snapshot we have) https://fabricweb.z5.web.core.windows.net/oufr/6.49.0/#examples/groupedlist

pressing left/right move focus up and down the tree, just like up/down arrows

**Comment by micahgodbolt** [comment]
The reason our examples go down an item when pressing left/right is that our example is wrapped with a focus zone missing direction

![image](https://user-images.githubusercontent.com/1434956/45378107-6863a600-b5b1-11e8-8b2b-d2356bd81dcc.png)


I believe the expected behavior is that once you get to a header item, you can use left/right to move from the selection circle to the open/close button. If you fix that focuszone issue what you get is left/right doing nothing even though that header is wrapped with a horizontal focus zone

![image](https://user-images.githubusercontent.com/1434956/45378173-9a750800-b5b1-11e8-83e9-64b271a25632.png)

So i'm guessing those keyboard events aren't making it to that header focus zone.

OH! Note that if you REMOVE the focus zone in the example, you can move left/right to get to the collapse button, but you can't traverse up/down the list.

Curious why we don't automatically supply that wrapping focus and selection zone used in the example.

**Comment by micahgodbolt** [comment]
Did this work that way in the past? With that horizontal focus zone, it seems that left/right arrow keys always moved from selection circle to arrow button.

i.e. is this more of a feature request then? I agree it'd be nice, but it seems like a larger sized functionality change.

**Comment by betrue-final-final** [comment]
As there's no columns in the header there shouldn't need to be column navigation. Technically there might be, but not from an end user perspective.

**Comment by Vitalius1** [comment]
@betrue-final-final I agree with Kevin. If we make so on `LEFT | RIGHT` we just expand the group, what would be the focus trail and how would we reach the `Select all` checkbox?

**Comment by mikewheaton** [comment]
@betrue-final-final Could you clarify what the expected behavior is for a header with a select/unselect all button as in the screenshot above?

**Comment by betrue-final-final** [comment]
Space should always select the selectable item, and I think that needs to be available for the whole row. I believe that is how tree controls work.

**Comment by mikewheaton** [comment]
Just spoke with Ben about this and the spec is:
- Left arrow: Collapse the group
- Right arrow: Expand the group
- Space bar: Select or deselect the group

**Comment by dzearing** [comment]
@KevinTCoughlin just a ping on this. Is there an eta or plan for this?

**Comment by dzearing** [comment]
Setting for March tentatively.

**Comment by powerchelle** [comment]
@dzearing hey, this one is a P1 for AMC. What's the scoop?
